test/units/TEST-30-ONCLOCKCHANGE.sh

```diff
@@ -8,7 +8,13 @@ systemd-analyze log-level debug
 systemctl disable --now systemd-timesyncd.service
 
 timedatectl set-timezone Europe/Berlin
-timedatectl set-time 1980-10-15
+
+# Get current time and add a future offset (e.g., 1 year ahead)
+current_time=$(date)
+future_time_berlin=$(date -d "$current_time + 1 year" +"%Y-%m-%d %H:%M:%S")
+
+# Set system time to the calculated future time for Europe/Berlin
```

Comment:
The comment is trivial. And mentioning timezone here sounds spurious. Please drop.

Reply:
done